data/serializer/src/main/java/tech/pegasys/teku/api/schema/deneb/SignedBeaconBlockAndBlobSidecar.java

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
I don't think we need `SignedBeaconBlockAndBlobSidecar`

Reply:
You're right. Removed it (Replaced it by `SignedBlobSidecar`)